Effects/Filters/source/shaders/Grain.hx

```diff
@@ -0,0 +1,160 @@
+package shaders;
+
+import openfl.display.Shader;
+
+/**
+ * ...
+ * @author MrCdK
+ */
+class Grain extends Shader
+{
+	
+	@fragment var code = '
```

Comment:
No, shaders are a compile time feature, I haven't choose how to load them from a file yet.